tests/hazmat/primitives/test_dh.py

```diff
@@ -202,7 +221,7 @@ def test_convert_to_numbers(self, backend, with_q):
                           dh.DHPrivateKeyWithSerialization)
 
     def test_numbers_unsupported_parameters(self, backend):
-        params = dh.DHParameterNumbers(23, 2)
+        params = dh.DHParameterNumbers(21, 2)
```

Comment:
I would be happier if this had a comment, because someone git-blaming this (heck, _me_ git blaming this) would probably not stop to think about why this is different/better and why 23 suddenly works now.